hack/tools/go.mod

```diff
@@ -8,68 +8,77 @@ require (
 	github.com/aojea/sloppy-netparser v0.0.0-20210819225411-1b3bd8b3b975
 	github.com/cespare/prettybench v0.0.0-20150116022406-03b8cfe5406c
 	github.com/client9/misspell v0.3.4
-	github.com/golangci/golangci-lint v1.64.5
+	github.com/golangci/golangci-lint/v2 v2.1.2
```

Comment:
```suggestion
	github.com/golangci/golangci-lint/v2 v2.1.5
```

Reply:
... which is now also on https://golangci-lint.run/product/changelog. I looked earlier today and unless I missed it, it wasn't mentioned yet.

Reply:
It was in the changelog: the changelog is published before the release.
I think it's just a browser cache effect.